Makefile.PL

```diff
@@ -107,6 +107,16 @@ grpc_google_default_credentials_create(NULL);
 EOT
 ) and ($EXTRA_DEFINES .= " -DGRPC_GOOGLE_DEFAULT_CREDENTIALS_CREATE_HAS_1_ARG");
 
+check_lib(
+    %CHECKLIB_ARGS,
+    debug => 1,
```

Comment:
I didn’t mean to leave this `debug` parameter in the PR. Feel free to remove it or leave it, as you prefer.

Reply:
Released this fix as grpc-xs-0.37.